src/components/countries/scss/infos.scss

```diff
@@ -0,0 +1,156 @@
+.info-container {
+  margin-top: 170px;
+  width: 70%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 50px;
+}
+
+.table-infos {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  padding-left: 2rem;
+  height: 500px;
+}
+
+.date {
+  float: left;
+  margin-left: 1rem;
+  font-weight: bolder;
+  color: #f60;
+  font-family: 'lato', sans-serif;
```

Comment:
- [ ] **[OPTIONAL]** There are several repetitions of this property `font-family: 'lato', sans-serif;` in most of the classes. It would be a good practice to create a separate class and add this property then use that class in any element that requires this property. What do you think? 🤔